docs/reference/settings/data-frames-settings.asciidoc

```diff
@@ -0,0 +1,39 @@
+
+[role="xpack"]
+[[data-frames-settings]]
+=== Data frames settings in Elasticsearch
+++++
+<titleabbrev>{dataframe-transforms} settings</titleabbrev>
+++++
+
+You do not need to configure any settings to use {dataframe-transforms}. It is enabled by default.
+
+All of these settings can be added to the `elasticsearch.yml` configuration file. 
+The dynamic settings can also be updated across a cluster with the 
```

Comment:
Just for my education: does the update require restarting the nodes in the rolling fashion or is it truly 'dynamic' in the sense that the nodes will just pick up the new value and start using it right after the update?

Reply:
It is truly dynamic, transforms just won't recognize it until they are individually restarted (as the variable is set on start)